Library/Formula/postgresql.rb

```diff
@@ -72,13 +74,18 @@ def install
       ENV.append %w{CFLAGS LDFLAGS}, "-arch #{Hardware::CPU.arch_32_bit}"
     end
 
+    system "pkill postgres || true"
     system "./configure", *args
     system "make", "install-world"
   end
 
   def post_install
-    unless File.exist? "#{var}/postgres"
-      system "#{bin}/initdb", "#{var}/postgres"
+    if (var/"postgres").exist?
+      mv var/"postgres", var/"postgres-#{version}"
```

Comment:
What if postgres and postgres-9.4 already exist? Maybe a user installs 9.4, then uses an old version of the formula to install 9.3, and is switching back now.